core/build.gradle

```diff
@@ -13,6 +13,8 @@ dependencies {
   compileOnly dep.intellijCore
 
   implementation dep.kotlinStdLib
+  implementation "androidx.collection:collection:1.1.0"
```

Comment:
You can drop this dep if you want. It'll be pulled in transitively and then there's only one version to think about.